spec/javascript/containers/PodcastShowContainer.js

```diff
@@ -0,0 +1,63 @@
+import PodcastShowContainer from '../../../app/javascript/react/containers/PodcastShowContainer.js'
+import fetchMock from 'fetch-mock'
+import { mount } from 'enzyme'
+import React from 'react'
+import { Link } from 'react-router'
+import jasmineEnzyme from 'jasmine-enzyme'
+
+describe('PodcastShowContainer', () => {
+  let wrapper, podcast
+
+  beforeEach(() => {
+    jasmineEnzyme()
+
+    podcast = {
+      id: 1,
+      title: 'PodCat',
+      description: 'cool podcast',
+      url: 'http://google.com',
+      availability: 'yes',
+      creators: ['Tyler', 'Bob']
+    }
+
+    fetchMock.get('/api/v1/podcasts/1', {
+      status: 200,
+      body: podcast
+    })
+
+    wrapper = mount(
+      <PodcastShowContainer
+        params={{id: 1}}
+      />
+    );
+  });
+
+  afterEach(fetchMock.restore)
+
+  it('should have initial empty state', () => {
+    expect(wrapper.state()).toEqual({ podcast: {} })
+  })
+
+  it('should have full state after api call', (done) => {
+    setTimeout(() => {
+      expect(wrapper.state()).toEqual({podcast: podcast})
+      done()
+    }, 0)
+  })
+
+  it('should render show page contents after api call', (done) => {
+    setTimeout(() => {
+      let expectIncludes = (wrapper, value) => {
+        expect(wrapper.text().includes(value)).toEqual(true)
+      }
```

Comment:
if you want to use this wonderful helper method, would you mind defining it in your test helper to be shared across all of your tests? 

Or even better, you could make a expectIncludesHelper.js file that is then required into your test helper. That would be the best separation of concerns